### PR TITLE
devplan.md

### LINKED ISSUE #88: Translate DevPlan to Spanish
https://wiki.aragon.one/documentation/Development_Plan/

### PR DESCRIPTION
**Pull request by DeividSaenz**
DevPlan translated to spanish. Closes #88

### CODE REVIEW

File: docs/translations/spanish/devplan.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+<img src="../../images/logo_text_under_dark.png">
+# Plan de Desarrollo
+Versión 1.2
+
+<img src="../../images/development_plan/01.png">
+</center>
+
+## **Introducción**
+
+Aragon is todo lo que necesitas para manejar organizaciones (compañías, ONGs, fundaciones, proyectos de código abierto…) sobre el blockchain de Ethereum. Implementa funcionalidades como tabla de capitalización, transferencia de tokens, votaciones, roles, recaudación de fondos y contabilidad. Las organizaciones de Aragon son fácilmente personalizables. Además, se pueden extender a través de la instalación de módulos de terceros.
```
Comment (izqui) [inline docs/translations/spanish/devplan.md]: - Aragon **es**

File: docs/translations/spanish/devplan.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+Las organizaciones de Aragon podrán optar a unirse a la Red Aragon, la cual proveerá de servicios como la posibilidad de actualización o un sistema de corte de arbitraje descentralizado.
```
Comment (izqui) [inline docs/translations/spanish/devplan.md]: - La **Aragon Network** (no traducimos el nombre)

File: docs/translations/spanish/devplan.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+Para más información, por favor visita nuestra página web, nuestro Libro Blanco de Apps Descentralizadas (dapp) y nuestro Libro Blanco de la Red.
```
Comment (izqui) [inline docs/translations/spanish/devplan.md]: - Libro blanco -> whitepaper (sin traducir tampoco)

File: docs/translations/spanish/devplan.md [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+
+Este documento describe cómo Aragon Dev planea desarrollar la dapp de Aragon, la Red de Aragon y cómo Aragon Dev se posiciona en el contexto más amplio del Proyecto Aragon.
```
Comment (izqui) [inline docs/translations/spanish/devplan.md]: Aragon Network again

File: docs/translations/spanish/devplan.md [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+
+[Stampery](https://stampery.com)– la compañía más reciente de Luis – hizo que las marcas de tiempo (timestamps) en blockchain fueran accesibles y ha trabajado con instituciones como el gobierno de Estonia, Microsoft y Telefónica. Ha participado en TechCrunch Disrupt y ha sido apoyado por el billonario Tim Draper.
+
+Luis también ha sido reconocido en [30 Under 30 de Forbes](https://www.forbes.com/30-under-30-europe-2016/technology/#6662a3e4a4b3), así como [Innovator Under 35 de the
+MIT](http://www.innovatorsunder35.com/innovator/luis-cuende).
+
+Antes de construir Aragon, Luis y Jorge estaban en Silicon Valley tratando de resolver el problema de los trolls de las patentes con Unpatent. Después de darse cuenta lo rota que estaba la infraestructura de innovación – los trolls de patentes a la alza, Donald Trump ganando las elecciones, la burocracia comiéndose al emprendimiento – ellos decidieron enfocar todo su tiempo en construir la infraestructura sobre la cual se administrarán las nuevas compañías y organizaciones.
```
Comment (izqui) [inline docs/translations/spanish/devplan.md]: patent trolls o trolles de patentes

**Comment by izqui** [comment]
Some changes are needed to merge, some of them are in more parts of the document too, as soon as those are fixed, we will merge it!

**Comment by DeividSaenz** [comment]
Should be good now.

**Comment by DeividSaenz** [comment]
Hello @izqui 
How can I claim the bounty? Is there somewhere I can find out?

**Comment by izqui** [comment]
The blockchain is super congested right now and yesterday's transaction didn't occur :/ 

Here is another try https://etherscan.io/tx/0x99c9b300f466fa499e459e2136d47d043c8469962fdafec750f1d555ee2324a5

**Comment by DeividSaenz** [comment]
Thanks a lot! I hope I can contribute again in the near future. As I am a designer, I guess that will be further down the road.

**Pull request merged**